### PR TITLE
riscv64-elf-binutils 2.41 riscv64-elf-gcc 13.2.0 riscv64-elf-gdb 13.2 (new formulae)

### PR DESCRIPTION
**Pull request by mkroening**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This adds the following new formulae, derived from their `aarch64-elf-*` counterparts:
* `riscv64-elf-binutils`
* `riscv64-elf-gcc`
* `riscv64-elf-gdb`

### CODE REVIEW

**Comment by mkroening** [comment]
After applying the patch, I get

```
$ brew audit --new riscv64-elf-binutils riscv64-elf-gcc riscv64-elf-gdb riscv64-elf-gcc
  * Formulae should not require patches to build. Patches should be submitted and accepted upstream first.
Error: 1 problem in 1 formula detected.
```

The patch is from upstream and will be included in the next release and fixes building on Monterey and Big Sur.

Should I remove the patch and restrict the package to Ventura? Or is there a way to silence this warning? :)

**Comment by mkroening** [comment]
> > Should I remove the patch and restrict the package to Ventura? Or is there a way to silence this warning? :)
> 
> We can live with failing strict audits if they are justifiable.

That's great. Looking at the CI output, this should be fine then, right? :)

**Review by ZhongRuoyu — APPROVED** [review]
Looks good to me. Thanks for your contribution, @mkroening!

**Comment by mkroening** [comment]
@ZhongRuoyu, what is the process of moving forward with this? :)

**Comment by ZhongRuoyu** [comment]
Let's wait for another maintainer to review it. Appreciate your patience, @mkroening.

**Review by p-linnane — APPROVED** [review]

**Review by Pandapsy — COMMENTED** [review]
mkroening:riscv64-cross

**Pull request merged**
